tools/generate_md_table.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import argparse
+import os
+import os.path as osp
+
+import yaml
+from mmengine import Config
+
+from mmdeploy.utils import get_backend, get_task_type, load_config
+
+
+def parse_args():
+    parser = argparse.ArgumentParser(
+        description='from yaml export markdown table')
+    parser.add_argument('yml_file', help='input yml config path')
+    parser.add_argument('output', help='output markdown file path')
+    parser.add_argument(
+        '--backends',
+        nargs='+',
+        help='backends you want to generate',
+        default=[
+            'onnxruntime', 'tensorrt', 'torchscript', 'pplnn', 'openvino',
+            'ncnn'
+        ])
+    args = parser.parse_args()
+    return args
+
+
+def main():
+    args = parse_args()
+    assert osp.exists(args.yml_file), f'File not exists: {args.yml_file}'
+    output_dir, _ = osp.split(args.output)
+    if output_dir:
+        os.makedirs(output_dir, exist_ok=True)
+    header = ['model', 'task'] + args.backends
+    aligner = [':--'] * 2 + [':--'] * len(args.backends)
```

Comment:
@kaizhong2021 hi, could make center alignment starting from coloumn 3rd.
```
aligner = [':--'] * 2 + [':-:'] * len(args.backends)
```

Reply:
OK，teacher，I will fix it later